ui5/browser/view/codeeditor.fragment.xml

```diff
@@ -1,16 +1,16 @@
 <core:FragmentDefinition xmlns="sap.m" xmlns:core="sap.ui.core" xmlns:l="sap.ui.layout" xmlns:e="sap.ui.codeeditor" xmlns:u="sap.ui.unified">
   <l:Splitter orientation="Vertical">
-    <Toolbar>
-      <layoutData><l:SplitterLayoutData size="35px" resizable="false"/></layoutData>
-      <u:FileUploader change="onChangeFile"/>
-      <Button text="Save as..." tooltip="Save current file as..." press="onSaveAs" />
-      <Button text="Save" tooltip="Save current file" press="onSaveFile" enabled="false" />
-      <Button text="Run" icon="sap-icon://play" tooltip="Run Current Macro" press="onRunMacro" enabled="false" />
-    </Toolbar>
-    <e:CodeEditor
-            height="100%"
-            colorTheme="default"
-            type="c_cpp"
-            value="{/code}" />
-  </l:Splitter>
+  <Toolbar>
+    <layoutData><l:SplitterLayoutData size="35px" resizable="false"/></layoutData>
+    <u:FileUploader change="onChangeFile"/>
+    <Button id="test" text="Save as..." tooltip="Save current file as..." press="onSaveAs" />
+    <Button text="Save" tooltip="Save current file" press="onSaveFile" enabled="false" />
+    <Button text="Run" icon="sap-icon://play" tooltip="Run Current Macro" press="onRunMacro" enabled="false" />
+  </Toolbar>
+  <e:CodeEditor
+          height="100%"
+          colorTheme="default"
+          type="c_cpp"
+          value="{/code}" />
+</l:Splitter>
 </core:FragmentDefinition>
```

Comment:
Is this file used now?

Reply:
Not anymore i am suppressing it in my next commit